pkgs/applications/graphics/displaycal/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, python3
+, xorg
+, argyllcms
+, wrapGAppsHook
+, gtk3
+, librsvg
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "displaycal";
+  version = "3.9.10";
+  format = "setuptools";
+
+  src = python3.pkgs.fetchPypi {
+    pname = "DisplayCAL";
+    inherit version;
+    hash = "sha256-oDHDVb0zuAC49yPfmNe7xuFKaA1BRZGr75XwsLqugHs=";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gtk3
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
```

Comment:
```
       error: undefined variable 'PyChromecast'

       at /Users/atemu/.cache/nixpkgs-review/pr-218309-13/nixpkgs/pkgs/applications/graphics/displaycal/default.nix:32:5:

           31|     distro
           32|     PyChromecast
             |     ^
           33|     send2trash
```



```Suggestion
  propagatedBuildInputs = with python3Pkgs; [
```

Reply:
Please fixup, this currently breaks eval/nixpkgs-review.

Reply:
There is no `python3Pkgs`. There is `python310Pkgs`. I just ran a `nixpkgs-review` successfully @ https://github.com/NixOS/nixpkgs/pull/219584#pullrequestreview-1325241857 and I'm on a commit with these changes. Can you give more info because this change doesn't even compile because of `python3Pkgs`?

Reply:
I just build it just fine directly from master.

Reply:
evals ok `nix-repl> displaycal
«derivation /nix/store/60kf5hp91k1kh5wbhc246hnldlk32rz7-displaycal-3.9.10.drv»`

python3.pkgs is the same thing as python3Packages

Reply:
Odd, I cannot reproduce the issue myself with manual evals either. Could you try `nixpkgs-review pr 218309`? It should fail after the git merge command.

Reply:
Alright, narrowed it down to something else. Really looked like this was the cause though oddly. Yay intransparent error messages?